plugins/woocommerce/changelog/52655-update-settings-gutenberg-components-edit-site

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Remove exported Gutenberg components from `@woocommerce/product-editor`
```

Comment:
Is this changelog necessary? If so, should it match changes with ./plugin/woocommerce? e.g. `plugins/woocommerce/package.json`

Reply:
Good call, changed this to a changelog comment instead in https://github.com/woocommerce/woocommerce/pull/52655/commits/135f750863c2715ca682740be35911e105c39543